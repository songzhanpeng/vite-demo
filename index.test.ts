import { it, expect } from 'vitest'
import { reverse } from './index'

it('反转字符串', () => {
  expect(reverse("test")).toMatchSnapshot()
})