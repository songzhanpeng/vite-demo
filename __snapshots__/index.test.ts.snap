// Vitest Snapshot v1, https://vitest.dev/guide/snapshot.html

exports[`反转字符串 1`] = `"tset"`;
